### PR TITLE
Add module_utils/dnos6.py

### PR DESCRIPTION
**Pull request by skg-net**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME
- modules/core/network/dnos6/dnos6_command.py
  ##### ANSIBLE VERSION
  lib/ansible/modules/core: (detached HEAD db38f0c876) last updated 2016/09/08 17:44:58 (GMT -700)
  lib/ansible/modules/extras: (detached HEAD 8bfdcfcab2) last updated 2016/09/08 17:44:58 (GMT -700)
  config file =
  configured module search path = Default w/o overrides
  ##### SUMMARY
  Added support for dnos6_command module, to support Dell Networking OS6 device

### CODE REVIEW

**Comment by skg-net** [comment]
Hi @privateip @gundalow, Please take a look at it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ Ansible Changes By Release
   * dnos10_command
   * dnos10_config
   * dnos10_template
+- dnos6
+  * dnos6_command
+
```
Comment (gundalow) [inline CHANGELOG.md]: Please remove blank line

File: lib/ansible/utils/module_docs_fragments/dnos6.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    required: true
+  port:
+    description:
+      - Specifies the port to use when building the connection to the remote
+        device.
+    required: false
+    default: 22
+  username:
+    description:
+      - This value I(username) is used to authenticate the SSH session to the
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/dnos6.py]: As per other PRs, you don't need to repeat the option name.

File: lib/ansible/utils/module_docs_fragments/dnos6.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        remote device. If the value is not specified in the task, the
+        value of environment variable ANSIBLE_NET_USERNAME will be used instead.
+    required: false
+  password:
+    description:
+      - This value I(password) is used to authenticate the SSH session to
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/dnos6.py]: As per other PRs, you don't need to repeat the option name.

File: lib/ansible/utils/module_docs_fragments/dnos6.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  password:
+    description:
+      - This value I(password) is used to authenticate the SSH session to 
+        the remote device. If the value is not specified in the task, the
+        value of environment variable ANSIBLE_NET_PASSWORD will be used instead.
+    required: false
+    default: null
+  ssh_keyfile:
+    description:
+      - This value I(ssh_keyfile) is the path to the key used to authenticate
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/dnos6.py]: As per other PRs, you don't need to repeat the option name.

**Comment by nitzmahone** [comment]
(ephemeral Windows test failure is ignorable, this is clearly not a Windows-affecting PR)

**Comment by skg-net** [comment]
ready_for_review

**Comment by mattclay** [comment]
@skg-net I've restarted the build. It appears all of the test failures were temporary and unrelated to your PR.

File: CHANGELOG.md [hunk 5]
```diff
@@ -46,6 +46,8 @@ Ansible Changes By Release
   * dnos10_command
   * dnos10_config
   * dnos10_template
+- dnos6
```
Comment (gundalow) [inline CHANGELOG.md]: This list should be in alphabetical order (dnos6 should be before dnos10)

**Comment by gundalow** [comment]
Minor fix and rebase needed

File: CHANGELOG.md (hunk 5 above)
Comment (skg-net) [reply CHANGELOG.md]: I thought since 1 < 6 its in alphabetical order, let me know if you want to change it.

**Comment by gundalow** [comment]
Well numeric order, so dnos6 comes before dnos10. 

Your rebate seems to have pulled in a load of other commits,  can you please resolve that.

**Comment by privateip** [comment]
Looks like you have done a merge commit here.  Please clean up the PR and rebase with devel before we can accept this PR

**Comment by skg-net** [comment]
@privateip @gundalow. I have completed the rebase, please take a look at it.
ready_for_review

**Pull request merged**